DataFormats/TauReco/src/PFRecoTauChargedHadron.cc

```diff
@@ -93,25 +99,28 @@ void PFRecoTauChargedHadron::print(std::ostream& stream) const
   if ( chargedPFCandidate_.isNonnull() ) {
     stream << " (" << chargedPFCandidate_.id() << ":" << chargedPFCandidate_.key() << "):"
 	   << " Pt = " << chargedPFCandidate_->pt() << ", eta = " << chargedPFCandidate_->eta() << ", phi = " << chargedPFCandidate_->phi() 
-	   << " (type = " << getPFCandidateType(chargedPFCandidate_->particleId()) << ")" << std::endl;
+	   << " (pdgId = " << chargedPFCandidate_->pdgId() << ")" << std::endl;
   } else {
     stream << ": N/A" << std::endl;
   }
   stream << "reco::Track: ";
   if ( track_.isNonnull() ) {
     stream << "Pt = " << track_->pt() << " +/- " << track_->ptError() << ", eta = " << track_->eta() << ", phi = " << track_->phi() << std::endl;
+  } else if ( lostTrackCandidate_.isNonnull() ) {
```

Comment:
Can you explain quickly what this addition means and why we did not have it before? In case there is no tack there is a lostTrack?